client/src/components/Overview/ImageGalleryThumbnail.jsx

```diff
@@ -5,12 +5,15 @@ var ImageGalleryThumbnail = function(props) {
     return (
       <React.Fragment>
         <div className='image-gallery-thumbnail'>
+          <img src={props.thumbnail_url} className="ig-thumbnail" />
```

Comment:
One thing I'm noticing, that you cuold use if you want, is to destructure your props at the top of your component. So instead of `var ImageGalleryThumbnail = function(props) {` it could be `var ImageGalleryThumbnail = function({ thumbnail_url } {`. This would let you call these variables without the `props.` prefacing them all. Though I get it if the props makes it more clear where they are coming from

Reply:
if you pass in some info about the image, you could add an `alt` tag to these for accessibility